use-cases/using-Intel-QAT-and-kata.md

```diff
@@ -0,0 +1,484 @@
+chipset by executing the following.
+
+### Identify which PCI Bus the Intel QAT card is on
+
+```sh
+$ for i in 0434 0435 37c8 1f18 1f19; do lspci -d 8086:$i; done
+```
+
+### Install necessary bundles for Clear Linux and disable kernel updates
+
+The Clear Linux installer generally installs the `os-core`, `os-core-update`, 
+and `openssh-server` bundle. Add the following minimum bundles including the 
+Linux kernel headers to complete the rest of this guide. The Linux kernel 
+headers are needed to compile the out of tree kernel modules to support Intel 
+QAT on the host. The `os-clr-on-clr` is a large development bundle that can be 
+installed instead to include everything below with additional items. This 
+guide installs the Linux LTS kernel version `4.19` from 2018 because the 
+crypto modules are set to `m` and not `y` which allows them to be replaced 
+with the out of tree driver. Otherwise you will need to build your own Clear 
+Linux kernel with the QAT drivers compiled in.
+
+```sh
+$ sudo swupd autoupdate --disable
+$ sudo swupd bundle-add network-basic kernel-lts2018 linux-lts-dev linux-dev make c-basic go-basic containers-virt dev-utils devpkg-elfutils devpkg-systemd
+$ sudo systemctl start docker
+$ sudo systemctl enable docker
+```
+
+### Replace Clear Linux kernel with LTS kernel
+
+```sh
+$ lts_kernel=$(sudo clr-boot-manager list-kernels | grep lts2018 | head -n 1)
+$ sudo clr-boot-manager set-kernel $lts_kernel
+$ sudo reboot
+```
+
+### Allow insecure modules to load to host
+
+```sh
+$ sudo mkdir -p /etc/kernel/cmdline.d
+$ cat << EOF | sudo tee /etc/kernel/cmdline.d/load-modules.conf
+module.sig_unenforce
+EOF
+$ sudo clr-boot-manager update
+$ sudo reboot
+```
+
+### Download Intel QAT drivers
+
+This will download the Intel QAT drivers from [01.org](https://01.org/intel-quickassist-technology). 
+Make sure to check the website for the latest version
+
+```sh
+$ mkdir -p $QAT_SRC
+$ cd $QAT_SRC
+$ curl -O $QAT_DRIVER_URL
+$ QAT_DRIVER_NAME=$(echo $QAT_DRIVER_URL | rev | cut -d"/" -f1 | rev)
+$ tar xzvf $QAT_DRIVER_NAME
+```
+
+### Compile Intel QAT drivers against host kernel 
+
+The Linux kernel headers bundle `linux-dev` should already have been added. 
+This replaces any existing modules provided by Clear Linux. If the Clear Linux
+kernel is updated, these modules must be rebuilt against that kernel and 
+reinstalled. By doing it this way you no longer have to rely on your 
+distribution to include the Intel QAT firmware. The modules will load every
+reboot and the qat_service helper script gets added. 
+
+```sh
+$ cd $QAT_SRC
+$ ./configure --disable-qat-lkcf  
+$ sudo -E make install -j$(nproc)
+$ sudo chown -R $USER:$USER $QAT_SRC
+```
+
+### Copy Intel QAT configuration files and enable Virtual Functions
+
+Learn more about customizing configuration files at the 
+[Intel QAT Engine github](https://github.com/intel/QAT_Engine/#copy-the-correct-intel-quickassist-technology-driver-config-files)
```

Comment:
The spell checker barfs here as the correct capitalisation is GitHub. But frankly, why not just delete the word as hovering over the link will make it obvious?

Reply:
> @eadamsintel - Travis is failing on the spell checker. That actually makes me happy since:
> 
> * we know it's actually working.
> * it's suggesting you replace "org" with "orgy" or "Borg" 👽 😄

That is weird because I don't have org anywhere in my guide as a single word. The closest is 01.org.